python/hail/docs/hailpedia/matrix_table.rst

```diff
@@ -0,0 +1,414 @@
+    +---------------+------------+---------------+
+    | locus         | alleles    | replicate_num |
+    +---------------+------------+---------------+
+    | locus<GRCh37> | array<str> |         int32 |
+    +---------------+------------+---------------+
+    | 20:10019093   | ["A","G"]  |             1 |
+    | 20:10019093   | ["A","G"]  |             2 |
+    | 20:10026348   | ["A","G"]  |             1 |
+    | 20:10026348   | ["A","G"]  |             2 |
+    | 20:10026357   | ["T","C"]  |             1 |
+    | 20:10026357   | ["T","C"]  |             2 |
+    | 20:10030188   | ["T","A"]  |             1 |
+    | 20:10030188   | ["T","A"]  |             2 |
+    | 20:10030452   | ["G","A"]  |             1 |
+    | 20:10030452   | ["G","A"]  |             2 |
+    +---------------+------------+---------------+
+
+Aggregation
+===========
+
+:class:`.MatrixTable` has three methods to compute aggregate statistics.
+
+- :meth:`.MatrixTable.aggregate_rows`
+- :meth:`.MatrixTable.aggregate_cols`
+- :meth:`.MatrixTable.aggregate_entries`
+
+These methods take an aggregated expression and evaluate it, returning
+a Python value.
+
+An example of querying entries is to compute the global mean of field `GQ`:
+
+    >>> mt.aggregate_entries(hl.agg.mean(mt.GQ))
+    67.73196915777027
+
+It is possible to compute multiple values simultaneously by
+creating a tuple or struct. This is encouraged, because grouping two
+computations together is far more efficient by traversing the dataset only once
+rather than twice.
+
+    >>> mt.aggregate_entries((agg.stats(mt.DP), agg.stats(mt.GQ)))
+    (Struct(mean=41.83915800445897, stdev=41.93057654787303, min=0.0, max=450.0, n=34537, sum=1444998.9999999995),
+    Struct(mean=67.73196915777027, stdev=29.80840934057741, min=0.0, max=99.0, n=33720, sum=2283922.0000000135))
+
+See the :ref:`sec-aggregators` page for the complete list of aggregator
+functions.
+
+Group-By
+========
+
+Matrix tables can be aggregated along the row or column axis to produce a new
+matrix table.
+
+- :meth:`.MatrixTable.group_rows_by`
+- :meth:`.MatrixTable.group_cols_by`
+
+First let's add a random phenotype as a new column field `case_status` and then
+compute statistics about the entry field `GQ` for each grouping of `case_status`.
+
+    >>> mt_ann = mt.annotate_cols(case_status = hl.cond(hl.rand_bool(0.5),
+    ...                                                 "CASE",
+    ...                                                 "CONTROL"))
+
+Next we group the columns by `case_status` and aggregate:
+
+    >>> mt_grouped = (mt_ann.group_cols_by(mt_ann.case_status)
+    ...                 .aggregate(gq_stats = agg.stats(mt_ann.GQ)))
+
+    >>> print(mt_grouped.entry.dtype.pretty())
+    struct {
+        gq_stats: struct {
+            mean: float64,
+            stdev: float64,
+            min: float64,
+            max: float64,
+            n: int64,
+            sum: float64
+        }
+    }
+
```

Comment:
each of these expressions has an extra newline between it producing separate boxes, which is fine, but I think it looks nicer all together.